tools/slang-unit-test/unit-test-coverage-tracing-metadata.cpp

```
@@ -251,6 +299,21 @@ SLANG_UNIT_TEST(coverageTracingMetadata)
+            slang_writeCoverageManifestJson(spirvBundle.coverage, manifest.writeRef()) == SLANG_OK);
+        SLANG_CHECK(manifest != nullptr);
+        UnownedStringSlice json(
+            (const char*)manifest->getBufferPointer(),
+            manifest->getBufferSize());
+        SLANG_CHECK(json.indexOf(toSlice("\"format\": \"slang-coverage\"")) != -1);
+        SLANG_CHECK(json.indexOf(toSlice("\"version\": 2")) != -1);
+        SLANG_CHECK(json.indexOf(toSlice("\"counter_count\"")) != -1);
+        SLANG_CHECK(json.indexOf(toSlice("\"space\"")) != -1);
+        SLANG_CHECK(json.indexOf(toSlice("\"binding\"")) != -1);
+        SLANG_CHECK(json.indexOf(toSlice("\"uniform_offset\"")) == -1);
+        SLANG_CHECK(json.indexOf(toSlice("\"uniform_stride\"")) == -1);
```

Cppcheck (2.20.0)

[style] The function 'FloatToFloatE5M2' is never used.

(unusedFunction)